content/en/docs/contribute/new-content/contributing-content.md

```diff
@@ -0,0 +1,267 @@
+---
+title: Opening a pull request
```

Comment:
nit: Capitalize all words for titles

Reply:
In both cases, the name and email you provide must match those found in your `git config`, and that your git name and email match those used for the CNCF CLA.

Reply:
@celestehorgan , OK. I don't have an opinion about the case of title text -- just in favor of consistency. There seemed to be widespread use of capitalizing titles throughout the docs; headings always use sentence case.